README.md

```diff
@@ -18,10 +18,8 @@ poetry run python vaccine/worker.py
 
 To run autoformatting and linting, run
 ```bash
-poetry run black .
-poetry run isort .
-poetry run mypy .
-poetry run flake8
+poetry run ruff check
+poetry run ruff format
```

Comment:
I think these (and any other `poetry` references in the README need to change to `uv`.